en,zh: Fix broken link and modify description about TiDBMonitor (#1369)
pingcap/docs-tidb-operator: en/monitor-a-tidb-cluster.md

### Set kube-prometheus and AlertManager

Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. 

To view these monitoring metrics in TidbMonitor Grafana, take the following steps:

1. Deploy Kubernetes cluster monitoring manually.

>    There are multiple ways to deploy Kubernetes cluster monitoring. If you are using kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).

Reviewer: ```suggestion
   There are multiple ways to deploy Kubernetes cluster monitoring.  To use kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).
```

### Set kube-prometheus and AlertManager

> There are NodeInfo and PodInfo dashboards in TidbMonitor Grafana, in order to obtain the monitoring metrics on Kubernetes. If you need, you should deploy Kubernetes monitor manually, reference [kube-prometheus](https://github.com/coreos/kube-prometheus), then configure `TidbMonitor.Spec.kubePrometheusURL` to obtain kube-prometheus metrics.

Reviewer: ```suggestion
Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. To use TidbMonitor Grafana to view these monitoring metrics, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus) to manually deploy Kubernetes cluster monitoring, and then set the ` TidbMonitor.spec.kubePrometheusURL` to get Kubernetes monitoring data.
```

> Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. To use TidbMonitor Grafana to view these monitoring metrics, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus) to manually deploy Kubernetes cluster monitoring, and then set the `TidbMonitor.spec.kubePrometheusURL` to obtain Kubernetes monitoring data.

Reviewer: ```suggestion
Nodes-Info and Pods-Info monitoring dashboards are built into TidbMonitor Grafana by default to view the corresponding monitoring metrics of Kubernetes. 

To view these monitoring metrics in TidbMonitor Grafana, take the following steps:

1. Deploy Kubernetes cluster monitoring manually.

   There are multiple ways to deploy Kubernetes cluster monitoring. If you are using kube-prometheus for deployment, see the [kube-prometheus documentation](https://github.com/coreos/kube-prometheus).

2. Set the `TidbMonitor.spec.kubePrometheusURL` to obtain Kubernetes monitoring data.
```
